docs/source/platform/schema-validation.mdx

```diff
@@ -3,27 +3,52 @@ title: Validating schema changes
+[//]: # (Description: Changes to this page that explain, in parallel with the monolithic version, how schema validation works with federation, highlighting that there's an extra step in the `apollo service:check` command which will build the composed schema to run traffic validation against. Ideally, this change could be done as a toggle either for each section or for the entire page where the extra details and steps are added for federation. If the toggle is difficult, then we should simply have a pattern that we use, like using a different color or an italicized font for the parts of the process that are federation-specific)
+[//]: # (Assignee: Caydie)
+[//]: # (Reviewer: Jake)
+
+There are many types of schema changes that can be potentially breaking to clients, like removing a field, if made without special consideration. For safety, some organizations take the approach of _never_ making these types of changes, but this leads to an ever-growing schema and reduced API flexibility over time. In the case of federated services where multiple organizations or teams are working on different parts of the same graph, these kinds of breaking changes are especially critical to monitor. In reality, making these types of changes to a schema can be very safe as long as you have tools in place to ensure that no queries are broken in the process.
 
 Apollo provides a tool to protect for exactly this scenario called **schema validation**.
 
+// TODO modify this top-level description for how schema validation for federation is done at the **service** level
+
 > **Note:** Schema validation is an Apollo Platform feature available on the [Team and Enterprise plans](https://www.apollographql.com/plans/) of [Apollo Engine](https://engine.apollographql.com).
 
+
 ## How it works
 
 Schema validation is run through the Apollo CLI by executing the `apollo service:check` command. Apollo will generate a diff between your local schema and your most recently registered schema, then validate that the changes are safe by checking if any queries actively running against your graph will be affected.
+If your graph is composed using federation, you can also use the `apollo service:check` command to validate individual service changes by providing a service name. However, unlike running service check on a non-federated service, Engine first needs to run composition on the proposed partial schema with the current list of implementing services and their partial schemas. Composition must pass before validation can continue.
 
 Here's how it works:
 
+// [TODO: Hopefully make this a toggle, where you can see the federation and the non-federation version of how it works].
```

Comment:
Let's forgo the toggle idea for now since adding in interactivity should align with our design paradigms that we're still trying to nail down.

Reply:
okie dokie :)